Add some additional info about plugins (#1221)
btcpayserver/btcpayserver-doc: docs/Development/Plugins.md

  |_ BTCPayServer.Plugins.Template
```

> Before starting, rename `BTCPayServer.Plugins.Template` to the name of your plugin of your chose.

Reviewer: ```suggestion
Before starting, rename `BTCPayServer.Plugins.Template` to the name of your plugin.
```

<link href="/Resources/css/my.css" asp-append-version="true" rel="stylesheet" />
```

> A good example of this is the [Bitcoin Whitepaper plugin](https://github.com/Kukks/BTCPayServerPlugins/tree/master/Plugins/BTCPayServer.Plugins.BitcoinWhitepaper) which expose the bitcoin whitepaper pdf on your BTCPayServer using embedded resources.

Reviewer: ```suggestion
A good example of this is the [Bitcoin Whitepaper plugin](https://github.com/Kukks/BTCPayServerPlugins/tree/master/Plugins/BTCPayServer.Plugins.BitcoinWhitepaper) which exposes the bitcoin whitepaper PDF on your BTCPay Server using embedded resources.
```

![Plugin Builder: Create a new build](../img/plugins/plugin-builder-create-build.png)

The result will be a packaged version of your plugin in `prerelease` state.
A version in prerelease can be modified just by rebuilding your plugin in the plugin builder.

> You can browse the prereleased plugin list on any BTCPay Server by going to `Server Settings/Policies`, check `Show plugins in pre-release` and `Save`.

Reviewer: ```suggestion
You can browse the prereleased plugin list on any BTCPay Server by going to `Server Settings > Policies`, check `Show plugins in pre-release` and `Save`.
```

> Once you click on the `Release` button in the build's page, the package won't be in prerelease anymore and visibile by everybody. When the package isn't in prerelease, you won't be able to publish a new build with the same version. So you will need to bump the `<Version>` of your plugin in the csproj before publishing any new adjustment to your plugin.

Reviewer: ```suggestion
Once you click the `Release` button on the build page, the package won't be in prerelease anymore and it is visible to everyone. Once the package is released, you won't be able to publish a new build with the same version number. So you will need to bump the `<Version>` of your plugin in the csproj before publishing any new adjustment to your plugin.
```
